examples/blk/mmc/client.c

```diff
@@ -119,8 +162,13 @@ void notified(microkit_channel ch)
 
     }
 
-    print_some_data();
+    if (!KEEP_REPEATING_COMMANDS) {
+        print_some_data();
+    }
     ensure_data_is_correct();
```

Comment:
I'm confused, if `KEEP_REPEATING_COMMANDS` is false shouldn't you be skipping the `ensure_data_is_correct`?

Reply:
Nevermind, I didn't read enough of the code around this block.